src/pages/Profile.tsx

```diff
@@ -1,5 +1,19 @@
+import Box from "@mui/material/Box";
+import Stack from "@mui/material/Stack";
+
+const styles = {
+  root: {
+    width: "100%",
+  },
+  postListContainer: { padding: 2, width: "50%" },
+};
+
 const Profile = () => {
-  return <h1 style={{ margin: "auto" }}>Profile</h1>;
```

Comment:
Since this is getting overhauled in another PR I think we should remove these changes and leave this page the way it was before

Reply:
Not sure where I got this change from, will revert it